Source/carthage/Build.swift

```diff
@@ -224,24 +224,44 @@ public enum BuildPlatform {
 	/// Build only for watchOS.
 	case watchOS
 
-	/// The `Platform` corresponding to this setting.
-	public var platform: Platform? {
+	case Multiple([BuildPlatform])
```

Comment:
It seems weird to me to make this a `case` instead of using `[BuildPlatform]` in the caller. Is there a reason we can't do that?